test/util/bspNet/block.ts

```diff
@@ -25,6 +25,39 @@ export interface SealedBlock {
   extSuccess?: boolean;
 }
 
+export const extendFork = async (
```

Comment:
Amazing helper. Could use some docs on what it does and what to pass it. Or at least the wrapper in `test-api.ts`, which is the one we'll be using in tests.

Reply:
Wrapper is already there, it's how it's being called in the `reorg-proof.test` file. Added docs now though